aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -6,7 +6,6 @@ use crate::{
         AbstractResourceWriteOp, GroupWrite, InPlaceDelayedFieldChangeOp,
         ResourceGroupInPlaceDelayedFieldChangeOp, WriteWithDelayedFieldsOp,
     },
-    check_change_set::CheckChangeSet,
```

Comment:
Did we get rid of the trait here b/c there's only a single usage pattern?

Reply:
Yes, it is only used by configs and I think it was a trait to enable testing with mock implementation. I moved it out of squash + change set creation, so it is no longer needed, hence single use case. Instead, it is generic on change set interface, which is more important (we can also implement this trait even for storage change sets).